app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -6,6 +6,16 @@ class ManageIQ::Providers::Redhat::InfraManager::Vm < ManageIQ::Providers::Infra
 
   supports_not :migrate, :reason => _("Migrate operation is not supported.")
 
+  supports :reconfigure_disks do
+    if storage.blank?
+      unsupported_reason_add(:reconfigure_disks, _('storage is missing'))
+    elsif ext_management_system.blank?
```

Comment:
The 'supports :control' cover more extensive cases than required. However, I'll have to share some of this logic with with https://github.com/ManageIQ/manageiq/pull/10505 (add quick_stats)